crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -599,8 +598,8 @@ pub enum Opcode {
     //memory
     Load(ArrayId),
     Store(ArrayId),
-    Intrinsic(OPCODE), //Custom implementation of usefull primitives which are more performant with Aztec backend
-    Nop,               // no op
+    Intrinsic(builtin::Opcode), //Custom implementation of usefull primitives which are more performant with Aztec backend
```

Comment:
We should remove the comment re Aztec backend since Noir is backend agnostic

usefull -> useful

Reply:
Done